src/util/cuda_vector_matrix.cu

```diff
@@ -0,0 +1,25 @@
+#include <vector>
+//#include <micm/util/cuda_vector_matrix.cuh>
```

Comment:
please delete commented out code

Reply:
Good catch!  Forgot to re-add this but my intention was to force the `.cu` implementation to follow the signature in the header file that way it is shared between the `.cu` file and the `.cpp` files that include it.  If this isn't the case, I can remove this.